IaC/pmm.cd/JenkinsStack.yml

```diff
@@ -770,10 +770,6 @@ Resources:
               			  ssl_trusted_certificate     /etc/nginx/ssl/ca-certs.pem;
               			  ssl_dhparam                 /etc/nginx/ssl/dhparam.pem;
 
-              			  ssl_protocols TLSv1.2;
```

Comment:
We already have the following setting:
ssl_protocols TLSv1.2 TLSv1.3;

This configuration allows to use TLSv1.3 if the client supports it, but it will also fall back to TLSv1.2 if necessary for compatibility with older clients.